apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -472,6 +472,8 @@ public void testOrphanDirsDeletionJavaAPI() throws Exception {
       ops.listFiles(tbLoc, file -> true, true, matchingFilesBefore);
       boolean orphaned = ops.deleteOrphanDirectory(tbLoc, ".trash", timeThreshold);
       Assertions.assertTrue(orphaned);
+      orphaned = ops.deleteOrphanDirectory(tbLoc, ".trash", timeThreshold);
```

Comment:
Can we please add comment on what is being tested here? Just looking at the code 474 vs 476 looks unclear to me.